python/hail/methods/statgen.py

```diff
@@ -229,20 +229,19 @@ def impute_sex(call, aaf_threshold=0.0, include_par=False, female_threshold=0.2,
     return kt
 
 
-@typecheck(dataset=MatrixTable,
-           ys=oneof(expr_float64, sequenceof(expr_float64)),
+@typecheck(ys=oneof(expr_float64, sequenceof(expr_float64)),
            x=expr_float64,
            covariates=sequenceof(expr_float64),
            root=str,
            block_size=int)
-def linear_regression(dataset, ys, x, covariates=[], root='linreg', block_size=16) -> MatrixTable:
-    """For each row, test a derived input variable for association with
+def linear_regression(ys, x, covariates=[], root='linreg', block_size=16) -> MatrixTable:
+    """For each row, test an input variable for association with
     response variables using linear regression.
 
     Examples
     --------
 
-    >>> dataset_result = hl.linear_regression(dataset, [dataset.pheno.height], dataset.GT.n_alt_alleles(),
+    >>> dataset_result = hl.linear_regression([dataset.pheno.height], dataset.GT.n_alt_alleles(),
```

Comment:
If @tpoterba wants to pass kwargs in the tutorial, probably should here as well.